src/app/app-routing.module.ts

```diff
@@ -1,5 +1,6 @@
 import { NgModule } from '@angular/core';
 import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
+import { AboutComponent } from './about/about.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/personalmoney/app-supabase/pullRequest?prid=7587366)
```suggestion
import { AboutComponent } from "./about/about.component";
```